doc_source/how-iot.md

```diff
@@ -60,15 +60,15 @@ Perform the following steps to create and configure this IAM role:
 
    ```
    {
-      "Version":"2012-10-17",
-      "Statement":[
+      "Version": "2012-10-17",
```

Comment:
I am trying to understand why an insignificant whitespace would cause an issue? What's the tooling that causes this issue?

Reply:
Sorry about confusing, aws cli is not happy about indentation of `}` at L71.

I guess it's related to json decoder aws cli is using. Those whitespace was added by my json formatter, just trying to make it consistent with other json file in this doc.